README.adoc

```diff
@@ -756,7 +753,14 @@ Use the following command to run the integration tests against your running clus
 include::{common-includes}/os-tabs.adoc[]
 [.tab_content.windows_section]
 --
-Substitute `[system-route-hostname]` and `[inventory-route-hostname]` with the appropriate values found by running the `oc get routes` command
+Run the following command, noting the values of the `system` and `inventory` route hostnames:
+
+[role=command]
+```
+oc get routes
+```
+
+Substitute `[system-route-hostname]` and `[inventory-route-hostname]` with the appropriate values in the following command:
```

Comment:
```suggestion
Substitute `[system-route-hostname]` and `[inventory-route-hostname]` with the appropriate values and run the following command:
```

Reply:
The values are taken from the command that follows, the change implies that the substitution should occur before running the command that displays the substitution values.

Reply:
I do not understand this.  If it is what you are saying -- i.e. run command to obtain values, substitute and then, execute another command -- we shall rework the section.

Reply:
My mistake, I thought this was a previous section. You're right, committing the change